engines/griffon/resources.cpp

```diff
@@ -293,26 +293,31 @@ void GriffonEngine::loadMap(int mapnum) {
 					rcDest.setWidth(16);
 					rcDest.setHeight(16);
 
+					int flag = 0;
 					if (l == 2 && curtilel == 1) {
 						for (int ff = 0; ff <= 5; ff++) {
 							int ffa = 20 * 5 - 1 + ff * 20;
 							int ffb = 20 * 5 + 4 + ff * 20;
 							if (curtile > ffa && curtile < ffb) {
-								_tiles[curtilel]->setAlpha(128, true);
+								flag = 1;
 							}
 						}
 					}
 					if (l == 1 && curtilel == 2) {
 						for (int ff = 0; ff <= 4; ff++) {
 							int ffa = 20 * (5 + ff) + 3;
 							if (curtile == ffa) {
-								_tiles[curtilel]->setAlpha(192, true);
+								flag = 2;
 							}
 						}
 					}
-
-					_tiles[curtilel]->blit(*_mapBg, rcDest.left, rcDest.top, Graphics::FLIP_NONE, &rcSrc);
-					_tiles[curtilel]->setAlpha(255, true);
+					if(flag == 1){
```

Comment:
Formatting is still incorrect, no space before opening curly brace.

Reply:
Please, simplify this code by using `alpha = <vale>` instead of flag. Just think about this logically. You change flag only under certain conditions, and otherwise, you're using previous flag value. Exactly the same thing will be with `alpha` but you will avoid those three calls to `blit()` at the bottom.

Reply:
i see, i will modify it soon and fix my git usage  errors